1.5/Source/PawnStorages/PawnStorages/CompPawnStorage.cs

```diff
@@ -20,6 +20,8 @@ public class CompPawnStorage : ThingComp
     private Dictionary<int, int> pawnStoringTick;
     private string transformLabelCache;
 
+    public Rot4 Rotation = default;
```

Comment:
this needs saving in PostExpostData or it won't persist through reloads

Reply:
Done